packages/terra-site/package.json

```diff
@@ -80,7 +80,8 @@
     "terra-time-input": "^1.13.0",
     "terra-toggle": "^1.11.1",
     "terra-toggle-button": "^1.12.1",
-    "xfc": "^1.0.0"
+    "xfc": "^1.0.0",
+    "cerner-consumer-theme": "file:../../../cerner-consumer-theme"
```

Comment:
Let's remove this until we get a release for cerner-consumer-theme. Once we have that, we'll want to add this as a devDependency so travis CI can install without failing out as it won't be able to install this.

Reply:
Sorry. It was committed unintentionally. https://github.com/cerner/terra-core/pull/966/commits/10cf489704028579deffb25cd49febe52e742b82